src/SFA.DAS.LevyTransferMatching.Web/Views/Pledges/Applications.cshtml

```diff
@@ -16,139 +16,157 @@
-                                    @app.DisplayAmount
-                                </td>
-                                <td class="govuk-table__cell">@app.Duration months</td>
-                                <td class="govuk-table__cell">
-                                    <criteria-list application="@app"></criteria-list>
-                                </td>
-                                <td class="govuk-table__cell"><strong class="@app.Status.GetCssClassForSender()">@app.Status.GetLabelForSender()</strong></td>
-                            </tr>
+                            <form action="/accounts/@Model.EncodedAccountId/pledges/create/inform" method="get">
+                                <button class="govuk-button">
+                                    Create a new transfer pledge
+                                </button>
+                            </form>
                         }
-                    }
 
-                    else
-                    {
-                        <tr class="govuk-table__row" data-table-row="one">
-                            <td colspan="6" class="govuk-table__cell">No applications received yet</td>
+                        @if (Model.UserCanClosePledge)
+                        {
+                            <form action="/accounts/@Model.EncodedAccountId/pledges/@Model.EncodedPledgeId/close" method="get">
+                                <button class="govuk-button govuk-link-secondary govuk-button--warning accepted-journey-cancel-button no-margin-top">
+                                Close pledge
+                                </button>
+                            </form>
+                        }
+
+                    </div>
+                </div>
+            </div>
+            <form method="post" id="applications_actions_form">
+                <details class="govuk-details">
+                    <summary class="govuk-details__summary">
+                        <span class="govuk-details__summary-text">Information about the applications</span>
+                    </summary>
+                    <div class="govuk-details__text">
+                        <h2 class="govuk-heading-s govuk-!-margin-bottom-0">Estimated cost in this financial year</h2>
+                        <p>This is an estimate of the cost of training for each application in this financial year. It is based on the funding band maximum and the planned start date of training.</p>
+                    </div>
+                </details>
+                <div class="govuk-form-group" das-highlight-error-for="ApplicationsToReject" error-class="govuk-form-group--error" id="ApplicationsToReject">
+                <table class="govuk-table" aria-label="Pledge applications">
+                    <thead class="govuk-table__head">
+                        <tr class="govuk-table__row">
+                            <th scope="col" class="govuk-table__header"> </th>
```

Comment:
When the table has no checkboxes, it looks like it's padded on the left a little. Is it possible to fix this? @jamesmking might be able to help

Reply:
James has kindly helped and updated this HTML on the Applications page as it should be in the prototype.